Added New pages (#11)
DarikshaAnsari/in-toto-docs: content/en/ecosystem/integrations.md

Reviewer: Make a change similar to the one described above for adopter.md. That is create a directory named `integrations`, etc.

---
title: Integrations

> description: Discover repositories with in-toto integrations from our friends.

Reviewer: ```suggestion
description: Projects and products with in-toto integrations.
```

---

This section lists software systems, services, or platforms that integrate with the project to provide additional functionality, interoperability, or compatibility. These integrations enhance the project's capabilities and extend its usefulness across various ecosystems.

> | Integration Name | Logo | Description |

Reviewer: ```suggestion
| Organization |  | Description |
```
